factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/validation/schema/SchemaConflictException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2017-2020 factcast.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.factcast.store.pgsql.validation.schema;
+
+public class SchemaConflictException extends IllegalStateException {
+
+    public SchemaConflictException(String string) {
+        super(string);
+    }
+
+    private static final long serialVersionUID = 1L;
+
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4877539)